### ISSUE TITLE
Update manifest to explicitly list installation targets

### ISSUE THREAD

**Issue by fthkucuk** [cschleiden/azure-boards-estimate#88]
I'm using Azure DevOps Server 2019 RC1. I got an error when I install the extension. I already attached a screenshot.
![Capture](https://user-images.githubusercontent.com/46280608/64426310-e66c6700-d0b6-11e9-941d-77b0be5c2e89.PNG)

**Comment by cschleiden** [comment]
Please try upgrading to 2019 Update 1: https://docs.microsoft.com/en-us/azure/devops/server/release-notes/azuredevops2019u1?view=azure-devops

**Comment by fthkucuk** [comment]
> Please try upgrading to 2019 Update 1: https://docs.microsoft.com/en-us/azure/devops/server/release-notes/azuredevops2019u1?view=azure-devops

Yeah, I know it. The reason for the problem is api version.  You should change to 2019 update 1 on the market place
![Capture](https://user-images.githubusercontent.com/46280608/64446111-aae69280-d0e0-11e9-9e40-e688dc076d5d.PNG)

**Comment by cschleiden** [comment]
Good point, we could update to something like:
```
{
    "targets": [
        {
            "id": "Microsoft.VisualStudio.Services.Cloud"
        },
        {
            "id": "Microsoft.TeamFoundation.Server",
            "version": "[15.0,)"
        }
    ]
}
```
